### PR TITLE
added taxon images

### PR DESCRIPTION
**Pull request by Fanevanjanahary**

### CODE REVIEW

**Comment by dimasciput** [comment]
Hi @Fanevanjanahary can you also push the migration file?

**Comment by Fanevanjanahary** [comment]
Hi, @dimasciput  added it

**Comment by dimasciput** [comment]
@Fanevanjanahary can you also make sure the taxon images will show up in the taxon dashboard?

File: bims/static/js/views/detail_dashboard/taxon_detail.js [hunk 1]
```diff
@@ -517,56 +517,70 @@ define(['backbone', 'ol', 'shared', 'underscore', 'jquery', 'chartJs', 'fileSave
...
-                            $firstColumnDiv.append('<a target="_blank" href="' + media['references'] + '">' +
-                                '<img title="Source: ' + media['publisher'] + '" alt="' + media['rightsHolder'] + '" src="' + media['identifier'] + '" width="100%"/></a>');
+            var $rowWrapper = $('<div id="gbif-images-row" class="gbif-images-row row gbif-images-row-fsdd"></div>');
+             $.get({
+                 url: '/api/taxon-images/'+ self.taxonId,
+                 dataType: 'json',
+                 success: function (data) {
+                     if (data.length > 0) {
+                         data.forEach(function (image){
+                             var $firstColumnDiv = $('<div class="col-6" "></div>');
```
Comment (dimasciput) [inline bims/static/js/views/detail_dashboard/taxon_detail.js:535]:
```suggestion
                             var $firstColumnDiv = $('<div class="col-6"></div>');
```

File: bims/static/js/views/detail_dashboard/taxon_detail.js [hunk 2]
```diff
@@ -517,56 +517,70 @@ define(['backbone', 'ol', 'shared', 'underscore', 'jquery', 'chartJs', 'fileSave
...
+                             $firstColumnDiv.append('<a target="_blank" href="'+image['url']+'">' +
+                                '<img src="' + image['url'] + '" width="100%"/></a>');
                             $rowWrapper.append($firstColumnDiv);
-                        }
-                        $wrapper.append($rowWrapper);
-                        if (!mediaFound) {
+                            $fetchingInfoDiv.hide();
+                     });
+                          $wrapper.append($rowWrapper);
+                     }else {
```
Comment (dimasciput) [inline bims/static/js/views/detail_dashboard/taxon_detail.js:542]:
```suggestion
                     } else {
```

**Review by dimasciput — COMMENTED** [review]
Few suggestions

**Comment by dimasciput** [comment]
Great thanks @Fanevanjanahary

**Pull request merged**